tachyon/math/finite_fields/generator/ext_prime_field_generator/build_defs.bzl

```diff
@@ -122,7 +122,6 @@ def generate_fp4s(
     _generate_ext_prime_fields(
         name = name,
         degree = 4,
-        base_field_degree = 2,
```

Comment:
Previously, `Fp4` could only be constructed with 2 `Fp2`s. Now, it can be constructed with 4 `Fp1`s, so the base_field_degree must be specified.

Reply:
? You state that the `base_field_degree` must be specified. So why is it not specified here then?

Reply:
Since we can't expect how `Fp4` is constructed here, `base_field_degree` should be injected from the outside.

```bazel
generate_fp4s(
    name = "fq4",
    base_field = "Fq2",
    base_field_degree = 2, # Like here
    base_field_hdr = "tachyon/math/circle/stark/fq2.h",
    class_name = "Fq4",
    namespace = "tachyon::math::stark",
    non_residue = [
        "2",
        "1",
    ],
    deps = [":fq2"],
)
```

```bazel
generate_fp4s(
    name = "koala_bear4",
    base_field = "KoalaBear",
    base_field_degree = 1, # Like here
    base_field_hdr = "tachyon/math/finite_fields/koala_bear/koala_bear.h",
    class_name = "KoalaBear4",
    namespace = "tachyon::math",
    non_residue = ["3"],
    deps = [":koala_bear"],
)
```